ndb/src/google/cloud/ndb/query.py

```diff
@@ -339,14 +339,17 @@ def resolve(self, bindings, used):
             used (Dict[Union[str, int], bool]): A mapping of already used
                 parameters.
 
-        Raises:
-            NotImplementedError: Always. This is because the implementation
-                will rely on as-yet-unimplemented features in
-                :class:`~google.cloud.ndb.model.Property`.
+        Returns:
+            Union[~google.cloud.ndb.query.DisjunctionNode, \
+                ~google.cloud.ndb.query.FilterNode, \
+                ~google.cloud.ndb.query.FalseNode]: A node corresponding to
```

Comment:
How will the implementation below return `FalseNode`?  I don't see a test below that triggers one.

Reply:
@tseaver This matches the union of the return types of `Property._IN` and `Property._comparison`. A `FalseNode` will be created when used with an empty container, e.g. `MyModel.foo_prop.IN([])`.

Reply:
OK, then maybe add a test which does that?

Reply:
Done